packages/api/src/router/user.ts

```diff
@@ -1,8 +1,12 @@
+/* Dependencies */
 import { type User } from '@prisma/client';
 import { TRPCError } from '@trpc/server';
 import { z } from 'zod';
 
+/* tRCP Config */
 import { createTRPCRouter, protectedProcedure, publicProcedure } from '../trpc';
+/* Utils */
+import setTempThumbnail from '../utils/setTempThumbnail';
```

Comment:
para mantener consistencia, podriamos nombrar el archivo como: `functions.ts ` tal y como se esta haciendo en bots:

![Screenshot 2023-03-25 at 4 29 18 PM](https://user-images.githubusercontent.com/10075532/227743067-d8b03396-ed7f-4d7c-aeab-bc666106d625.png)

```suggestion
import { setThumbnailUrl } from '../utils/functions';
```